pkg/appmgmt/general/general_test.go

```diff
@@ -208,10 +210,12 @@ func TestOriginatorPod(t *testing.T) {
 	}
 	am.AddPod(&pod1)
 	assert.Equal(t, len(app.GetNewTasks()), 2)
-	task, err = app.GetTask("UID-POD-00002")
+	task, err = app.GetTask("UID-POD-00001")
```

Comment:
Hi @wilfred-s , if we always return the pod, the test here the originator will be the pod1 who first add into cluster.